Tasks/XcodeV5/xcode.ts

```diff
@@ -17,16 +17,18 @@ async function run() {
 
         let xcodeVersionSelection: string = tl.getInput('xcodeVersion', true);
         telemetryData.xcodeVersionSelection = xcodeVersionSelection;
-
+        
+        let devDir: string;
         if (xcodeVersionSelection === 'specifyPath') {
-            let devDir = tl.getInput('xcodeDeveloperDir', true);
+            devDir = tl.getInput('xcodeDeveloperDir', true);
             tl.setVariable('DEVELOPER_DIR', devDir);
         }
         else if (xcodeVersionSelection !== 'default') {
             // resolve the developer dir for a version like "8" or "9".
-            let devDir = utils.findDeveloperDir(xcodeVersionSelection);
+            devDir = utils.findDeveloperDir(xcodeVersionSelection);
```

Comment:
I don't think that it makes sense to track `xcodeDeveloperDir` if `xcodeVersionSelection != 'specifyPath'`.

Reply:
Fixed